### ISSUE TITLE
wunderground weather provider

### ISSUE THREAD

**Issue by sphh** [tmalch/lightningWeather#3]
Great idea!!

Would it be possible to add [Wunderground](https://www.wunderground.com) as a service provider?

**Comment by tmalch** [comment]
Yes should be possible, but I don't know when I'll find time for that.

**Comment by sphh** [comment]
Thanks for considering it!
